lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<EOM
+
+'#{METADATA_REPOSITORY}' repository not found.
+Try running the command:
+  git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git ../#{METADATA_REPOSITORY}
```

Comment:
That is a pretty specific command, when there are more than one commands that could work.  And they may not be running in the folder you are thinking.  It may risk some wrong moves here.

Would a better communication be to go to the `xruby/README.md` file?